### DISCUSSION TITLE
[Bug] ActiveSelection changes objects position

### DISCUSSION THREAD

**Discussion by xorb** [fabricjs/fabric.js#9084] [Q&A]
ActiveSelection moves objects randomly.  

Demo:
https://codesandbox.io/p/sandbox/fabric-artboard-forked-rxkvm4?file=%2Fsrc%2FApp.tsx%3A110%2C10

Steps to reproduce: 
1. Select multiple objects
2. Rotate active selection
3. Create a new selection either using pointer or using button: select all objects
 
[active-selection-issue.webm](https://github.com/fabricjs/fabric.js/assets/39509710/9ba062b6-8ca1-44a6-8217-cba8ad2dbc5a)

@ShaMan123 can you take a look ?

**Comment by ShaMan123** [comment]
This is really strange
I managed to reproduce in your repro ~~but not in [mine](https://codesandbox.io/s/fabric-vanillajs-sandbox-forked-wrn5dx?file=/src/index.ts:786-827) - please try on mine~~
Also if you first move the selection and then rotate the bug does not occur
Same if you move each rect before selection

**Comment by ShaMan123** [comment]
Let's continue this in #9087
